poc/vdaf_pine.py

```diff
@@ -471,6 +512,7 @@ def helper_proofs_share(self,
             k_share,
             self.domain_separation_tag(USAGE_PROOF_SHARE),
             byte(self.PROOFS) + byte(agg_id),
+            self.flp_squared_norm_equality.PROOF_LEN +
             self.flp.PROOF_LEN * self.PROOFS
```

Comment:
This is visually hard to parse: the two lines look like different input parameters. Here and below.
```suggestion
            self.flp_squared_norm_equality.PROOF_LEN +
                self.flp.PROOF_LEN * self.PROOFS
```